README.md

```diff
@@ -1,14 +1,14 @@
 ![](workshops/static/amy-logo.png)
 
-![](https://travis-ci.org/swcarpentry/amy.svg?branch=master)
+![](https://travis-ci.org/swcarpentry/amy.svg?branch=master) [![](https://img.shields.io/badge/python-3.4+-blue.svg)](https://www.python.org/downloads/release/python-340/)
```

Comment:
Did you check if you can use single newline instead of space between these badges and still display the badges in one line?